endpoints/data_objects/src/main.cpp

```diff
@@ -1273,14 +1282,24 @@ namespace
 					logging::trace(*_sess_ptr, "{}: Opening secondary output streams to [{}].", fn, lpath_iter->second);
 					for (int i = 0; i < stream_count; ++i) {
 						pw_streams.emplace_back(std::make_shared<parallel_write_stream>(
-							client_info.username, lpath_iter->second, openmode, ticket, &pw_streams.front()->stream()));
+							client_info.username,
+							lpath_iter->second,
+							std::nullopt,
+							openmode,
+							ticket,
+							&pw_streams.front()->stream()));
 					}
 				}
 				catch (const irods::exception& e) {
 					logging::error(*_sess_ptr, "{}: {}", fn, e.client_display_what());
-					logging::error(
-						*_sess_ptr, "{}: Could not open one or more output streams to [{}].", fn, lpath_iter->second);
-					res.result(http::status::internal_server_error);
+					// clang-format off
+					res.body() = json{
+						{"irods_response", {
+							{"status_code", e.code()},
+							{"status_message", e.client_display_what()}
+						}}
+					}.dump();
+					// clang-format on
```

Comment:
Pull this change into the same commit as this: https://github.com/irods/irods_client_http_api/pull/441/files#r2234023218